gem/styles/gem-springster/_base.scss

```diff
@@ -1,8 +1,6 @@
 html, body  {
```

Comment:
I've moved it to _m-typography.scss, also we're not using rems so adding it on the html tag seems unnecessary